### PR TITLE
Added Ubuntu 18.04

### PR DESCRIPTION
**Pull request by rhomson2**

### CODE REVIEW

File: provisioning/ubuntu1804_preseed.cfg [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+d-i user-setup/encrypt-home boolean false
+
+### Package selection
+tasksel tasksel/first standard
+d-i pkgsel/include string openssh-server build-essential open-vm-tools
+d-i pkgsel/install-language-support boolean false
+
+# disable automatic package updates
+d-i pkgsel/update-policy select none
+d-i pkgsel/upgrade select full-upgrade
```
Comment (karras) [inline provisioning/ubuntu1804_preseed.cfg]: Please add a new line at the end of the file :)

File: provisioning/ubuntu1804_preseed.cfg [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+d-i partman-md/confirm boolean true
+d-i partman-partitioning/confirm_write_new_label boolean true
+d-i partman/choose_partition select finish
+d-i partman/confirm boolean true
+d-i partman/confirm_nooverwrite boolean true
+
+### Account setup
+d-i passwd/user-fullname string installer
+d-i passwd/user-uid string 1000
+d-i passwd/user-password password rootpw
```
Comment (karras) [inline provisioning/ubuntu1804_preseed.cfg]: Please use hashed and not plaintext passwords

File: provisioning/ubuntu1804_preseed.cfg [hunk 3]
```diff
@@ -0,0 +1,61 @@
+# Preseeding only locale sets language, country and locale.
+d-i debian-installer/locale string en_US
+
+# Keyboard selection.
+d-i console-setup/ask_detect boolean false
+d-i keyboard-configuration/xkb-keymap select de-CH
```
Comment (karras) [inline provisioning/ubuntu1804_preseed.cfg]: Please switch to the `en` / `us` keyboard layout thus we are aligned with the other existing images

File: provisioning/ubuntu1804_preseed.cfg [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+d-i passwd/username string installer
+
+# The installer will warn about weak passwords. If you are sure you know
+# what you're doing and want to override it, uncomment this.
+d-i user-setup/allow-password-weak boolean true
+d-i user-setup/encrypt-home boolean false
+
+### Package selection
+tasksel tasksel/first standard
+d-i pkgsel/include string openssh-server build-essential open-vm-tools
```
Comment (karras) [inline provisioning/ubuntu1804_preseed.cfg]: Ideally remove the `open-vm-tools` as the target hypervisor for such images is not always VMWare

**Comment by karras** [comment]
As discussed, thanks a lot for the hard work! What do you think about my comments / proposals?

**Review by karras — CHANGES_REQUESTED** [review]

**Comment by rhomson2** [comment]
Added your propositions.
Also added PROXMOX Support for Ubuntu 18.04 and 20.04

**Review by karras — APPROVED** [review]

**Comment by karras** [comment]
Thanks :1st_place_medal:

**Pull request merged**